src/gRPC/hpo_pb2_grpc.py

```diff
@@ -30,8 +30,8 @@ def __init__(self, channel):
                 request_serializer=hpo__pb2.ExperimentDetails.SerializeToString,
                 response_deserializer=hpo__pb2.NewExperimentsReply.FromString,
                 )
-        self.StopExperiment = channel.unary_unary(
-                '/helloworld.HpoService/StopExperiment',
+        self.DeleteExperiment = channel.unary_unary(
```

Comment:
@kusumachalasani - We shouldn't update these files by hand rather generate them

Reply:
oh..ok! didn't know. Will check that. Thanks!

Reply:
You can use the below cmd, which can be found in the readme as well - https://github.com/kruize/hpo/tree/main/src#readme

```
python3 -m grpc_tools.protoc -I ./gRPC/protos --python_out=./gRPC --grpc_python_out=./gRPC gRPC/protos/*.proto
```